deploy-if-version-bump.sh

```diff
@@ -1,11 +1,20 @@
 #!/bin/sh
 
 set -e
-if ! git rev-parse `python setup.py --version` 2>/dev/null ; then
-    git tag `python setup.py --version`
-    git push --tags  # update the repository version
-    python setup.py bdist_wheel  # build this package in the dist directory
-    twine upload dist/* --username $PYPI_USERNAME --password $PYPI_PASSWORD  # publish
-else
-    echo "No deployment - Only non-functional elements were modified in this change"
+if [ $PY_VERSION = "2" ]; then
+  if ! git rev-parse `python setup.py --version` 2>/dev/null ; then
+      git tag `python setup.py --version`
+      git push --tags  # update the repository version
+      python setup.py bdist_wheel  # build this package in the dist directory
+      twine upload dist/* --username $PYPI_USERNAME --password $PYPI_PASSWORD  # publish
+  else
+      echo "No deployment - Only non-functional elements were modified in this change"
+  fi
+fi
+
+if [ $PY_VERSION = "3" ]; then
```

Comment:
So if the Python version is `3` then there is no functional changes check. Is that really the goal?